package.yaml

```diff
@@ -77,6 +77,7 @@ dependencies:
   - process == 1.6.*
   - safe == 0.3.*
   - singletons == 3.0.*
+  - singletons-base == 3.1.*
```

Comment:
Is this import now required? Was it just added for the type level list append experiments?